Dynatrace/dynatrace-operator
# Moves scripts to improve repository structure

# Description

This PR moves all shell scripts that I could find into the `./hack` folder or into a subfolder of `./hack`.
It improves the repository's structure and findability of said files.

## How can this be tested?

Setup a fresh cluster of your preferred cloud provider.  
Then, run all Makefile targets to make sure the paths match and everything is found.  
There is still the possibility that I overlooked something in the GitHub Actions workflow, which will become apparent after this PR has been opened.  

## Checklist
